source/postprocess/visualization/heating.cc

```diff
@@ -120,17 +125,31 @@ namespace aspect
 
             for (unsigned int c=0; c<this->n_compositional_fields(); ++c)
               in.composition[q][c] = uh[q][this->introspection().component_indices.compositional_fields[c]];
+
+            mid_point += evaluation_points[q]/n_quadrature_points;
           }
 
         in.position = evaluation_points;
 
-        this->get_material_model().evaluate(in, out);
+        typename DoFHandler<dim>::active_cell_iterator cell;
+        cell = (GridTools::find_active_cell_around_point<> (this->get_mapping(), this->get_dof_handler(), mid_point)).first;
```

Comment:
Alternatively, I think we wrote the interface to `material_model->evaluate()` so that it can cope with not getting a cell -- that's why we pass a pointer. So you could pass `NULL` instead. The question of course is whether your material model actually needs this.